spec/lib/bandiera/client_spec.rb

```diff
@@ -53,23 +53,64 @@
     end
   end
 
-  context 'when some RestClient::Resource options are passed' do
-    it 'passes them onto the RestClient::Resource' do
+  context 'when some Typhoeus::Request options are passed' do
+    it 'passes them onto the Typhoeus::Resource' do
       params   = {}
       options  = { timeout: 24, open_timeout: 24 }
-      response = double(:response, body: JSON.generate({ response: {} }))
-      resource = double(:resource, '[]' => double(get: response))
+      response = double(:response, body: JSON.generate({ response: {} }), success?: true)
+      resource = double(:resource, run: response)
 
-      expect(RestClient::Resource)
+      expect(Typhoeus::Request)
         .to receive(:new)
-        .with(api_uri, method: :get, timeout: 24, open_timeout: 24, headers: anything)
+        .with("#{api_uri}/v2/groups/foo/features/bar", method: :get, timeout: 24, open_timeout: 24, headers: {"User-Agent"=>"Bandiera Ruby Client / 3.0.4"}, params: {})
```

Comment:
Is there a way to auto-generate the user agent string?  As I can see this test failing when we bump the version...